### PR TITLE
fix: CET to UTC times

### PR DESCRIPTION
**Pull request by dipanshuraz**
#416 Work In Progress for time conversion.

### CODE REVIEW

**Review by bebatut — COMMENTED** [review]
Thanks for the contribution. It looks globally good.
I was just a doubt. I think from the last Sunday in March to the last Sunday in October, CET is UTC+1 not UTC+2.

File: _data/ols-1-schedule.yaml [hunk 1]
```diff
@@ -88,7 +88,7 @@ weeks:
         duration: 60 min
         title: Mentor training
         date: January 17, 2020
-        time: '17:00'
```
[the remark this answers is not in the document]
Comment (dipanshuraz) [reply _data/ols-1-schedule.yaml:91]: Thanks for detailed information 👍

File: _includes/timeline.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
     {% assign type = type | remove_first: ' + ' %}
 {% endif %}
 {% if e.date %}
-- **{{ e.date }}** {% if e.time %}([{{ e.time }} Universal Time](https://arewemeetingyet.com/Berlin/{{ e.date | date: "%Y-%m-%d" }}/{{ e.time }}/OLS%20Application%20Call)){% endif %}: {{ e.description }}{% if e.type %}({{ type }}){% endif %}{% if e.notes %} - <i class="fas fa-clipboard"></i> [Notes with Zoom call link]({{ e.notes }}){% endif %}{% if e.recording %} - <i class="fab fa-youtube"></i> [Recording]({{ e.recording }}){% endif %}{% if e.details %}
+- **{{ e.date }}** {% if e.time %}([{{ e.time }} Universal Time](https://arewemeetingyet.com/{{ e.date | date: "%Y-%m-%d" }}/{{ e.time }}/OLS%20Application%20Call)){% endif %}: {{ e.description }}{% if e.type %}({{ type }}){% endif %}{% if e.notes %} - <i class="fas fa-clipboard"></i> [Notes with Zoom call link]({{ e.notes }}){% endif %}{% if e.recording %} - <i class="fab fa-youtube"></i> [Recording]({{ e.recording }}){% endif %}{% if e.details %}
```
Comment (bebatut) [inline _includes/timeline.md:10]:
```suggestion
- **{{ e.date }}** {% if e.time %}([{{ e.time }} Universal Time](https://arewemeetingyet.com/UTC/{{ e.date | date: "%Y-%m-%d" }}/{{ e.time }}/OLS%20Application%20Call)){% endif %}: {{ e.description }}{% if e.type %}({{ type }}){% endif %}{% if e.notes %} - <i class="fas fa-clipboard"></i> [Notes with Zoom call link]({{ e.notes }}){% endif %}{% if e.recording %} - <i class="fab fa-youtube"></i> [Recording]({{ e.recording }}){% endif %}{% if e.details %}
```

File: _data/ols-1-schedule.yaml [hunk 3]
```diff
@@ -561,7 +561,7 @@ weeks:
         duration: 90 min
         title: 'Mental health care, Ally skills and Career Guidance Call'
         date: April 1, 2020
-        time: '19:00'
+        time: '18:00'
```
Comment (bebatut) [inline _data/ols-1-schedule.yaml:564]: I think here it would be again UTC+2 until end of october

**Comment by dipanshuraz** [comment]
CEST time for year - 

2020 : 29 March 2020 - 25 October 2020
2021 : 28 March 2021 - 31 October 2021
2022 : 27 March 2022 - 30 October 2022 
2023 : 26 March 2023 - 29 October 2023

**Review by pazbc — COMMENTED** [review]
I checked the  CET/CEST to UTC conversion and it is correct :)

**Comment by bebatut** [comment]
Thanks @dipanshuraz for that PR

**Pull request merged**